### DISCUSSION TITLE
If antiSMASH is suitable for plant genome

### DISCUSSION THREAD

**Discussion by zhangwenda0518** [antismash/antismash#603] [Q&A] [answered]
Hello, I'd like to ask if antiSMASH is suitable for plant genome.
I installed the docker image of Plantismash, but the version of antiSMASH there is 4.0, and there are many Cf_putative comments in my results, which are the annotation results of the old version of antiSMASH.
Can I use the latest version of antiSMASH with plantismash? Or do I just need to update the latest database?
Can you give me some advice on my plant analysis?
Thank you。

**Comment by SJShaw** [comment, marked as answer]
antiSMASH is not designed for plants at all, there's no reason to use it.
plantiSMASH (https://github.com/plantismash/plantismash) is a modified version of antiSMASH (somewhere between 3.0 and 4.0) intended specifically for plants, so you should use that for your plant analysis instead of antiSMASH.
